src/sql/workbench/browser/editor/profiler/profilerInput.ts

```diff
@@ -192,8 +198,10 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
 		return this._filter;
 	}
 
-	public onSessionStopped(notification: azdata.ProfilerSessionStoppedParams) {
-		this._notificationService.error(nls.localize("profiler.sessionStopped", "XEvent Profiler Session stopped unexpectedly on the server {0}.", this.connection.serverName));
+	public onSessionStopped(notification: azdata.ProfilerSessionStoppedParams, isFileSession: boolean = false) {
+		if (!isFileSession) {	// File session do not have serverName, so ignore notification error based off of server
+			this._notificationService.error(nls.localize("profiler.sessionStopped", "XEvent Profiler Session stopped unexpectedly on the server {0}.", this.connection.serverName));
```

Comment:
Although here we hardcode XEvent...am I incorrect in my assumption that the profiler is supposed to be a generic extensibility feature? If it's currently MSSQL specific then I guess it doesn't really matter if you reference XEL directly or not (although then I would ask why it exists in the extensibility layer at all)

Reply:
Just to reduce the overhead, I will keep this message as-is, even though hopefully in future it would be made extensible.